.github/workflows/pre-main.yml

```diff
@@ -28,3 +28,12 @@ jobs:
                 level: warning
               braces:
                 level: warning
+  deprecated-k8s-check:
+    name: Pluto Deprecated Version Check
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Download Pluto
+        uses: FairwindsOps/pluto/github-action@master
+      - name: Use pluto
+        run: |
+          pluto detect-files
```

Comment:
Should we use the -d option with the root directory of the repo. I tried pluto manually on the partner repo yamls and I go an output:
[deliedit@fedora pluto]$ ./pluto detect-files -d ~/partner
NAME                                             KIND                       VERSION                             REPLACEMENT                    REMOVED   DEPRECATED  
network-attachment-definitions.k8s.cni.cncf.io   CustomResourceDefinition   apiextensions.k8s.io/v1beta1        apiextensions.k8s.io/v1        true      true        
multus                                           ClusterRole                rbac.authorization.k8s.io/v1beta1   rbac.authorization.k8s.io/v1   true      true        
multus                                           ClusterRoleBinding         rbac.authorization.k8s.io/v1beta1   rbac.authorization.k8s.io/v1   true      true        
kube-multus-ds-amd64                             DaemonSet                  extensions/v1beta1                  apps/v1                        true      true        
kube-multus-ds-ppc64le                           DaemonSet                  extensions/v1beta1                  apps/v1                        true      true        
network-attachment-definitions.k8s.cni.cncf.io   CustomResourceDefinition   apiextensions.k8s.io/v1beta1        apiextensions.k8s.io/v1        true      true        
multus                                           ClusterRole                rbac.authorization.k8s.io/v1beta1   rbac.authorization.k8s.io/v1   true      true        
multus                                           ClusterRoleBinding         rbac.authorization.k8s.io/v1beta1   rbac.authorization.k8s.io/v1   true      true        
kube-multus-ds-amd64                             DaemonSet                  extensions/v1beta1                  apps/v1                        true      true        
kube-multus-ds-ppc64le                           DaemonSet                  extensions/v1beta1                  apps/v1                        true      true        
network-attachment-definitions.k8s.cni.cncf.io   CustomResourceDefinition   apiextensions.k8s.io/v1beta1        apiextensions.k8s.io/v1        true      true        
multus                                           ClusterRole                rbac.authorization.k8s.io/v1beta1   rbac.authorization.k8s.io/v1   true      true        
multus                                           ClusterRoleBinding         rbac.authorization.k8s.io/v1beta1   rbac.authorization.k8s.io/v1   true      true        
kube-multus-ds-amd64                             DaemonSet                  extensions/v1beta1                  apps/v1                        true      true        
kube-multus-ds-ppc64le                           DaemonSet                  extensions/v1beta1                  apps/v1                        true      true

Reply:
Yes, probably.  I'm actually not sure where it's pulling those YAMLs from that it's flagging because if you do all of the directories it doesn't show up with anything.

Reply:
Oh whoops, `/temp`.

Reply:
Modified it to specifically point at which folders to run the check against.

Reply:
Do you have a log for the issue ?